english/dream/lucidDream.md

```diff
@@ -0,0 +1,15 @@
+
+You spend a moment viewing all the details of this marshmallow prison 
+
+with complete clarity.  You discard the straight jacket simply by willing it. 
+
+Then decide, the best way to handle the situation is to 
+
+[eat your way out of it](../eating-walls/eating-marshmallows.md). 
+
+
+But then you remember that this is a lucid dream.  You can do whatever you want. 
+
+So, instead you decide to [fly](../fly/fly.md) until just before deciding to go to [nepal](../nepal/nepal.md).
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.